topiary-cli/tests/samples/expected/ocaml.ml

```diff
@@ -1415,3 +1415,57 @@ type foo = {
   baz:
     [`Qux | `Bar];
 }
+
+(* #914 to #917 Edge cases in dangling behaviours *)
+let _ =
+  match foo with
+  | _ ->
+    (
+      match bar with
+      | [] -> None
+      | _ -> foo
+    )
+
+let _ =
+  match foo with
+  | _ ->
+    (
+      if path = [] then
+          ()
+      else
+          ()
+    )
+
+let foo : (bar, baz) qux =
+    ()
```

Comment:
Are there meant to be double indents before `()` on lines 1434, 1436 and 1440?

Reply:
Oh, didn't even catch that one. I feel it might be related to my own comments: a bunch of places where there is one too many indent levels, either because we're expecting only one level (here) or zero (my comments above).